Cargo.toml

```diff
@@ -30,6 +29,7 @@ semver = "0.2.1"
 toml = "0.1"
 unicode-normalization = "0.1"
 quine-mc_cluskey = "0.2.2"
+clippy_lints = { version = "0.0.*", path = "clippy_lints" }
```

Comment:
we can add this to utils/update_lints.py so the version is automatically checked